pallets/avn-anchor/src/lib.rs

```diff
@@ -288,9 +319,47 @@ pub mod pallet {
 
             Ok(())
         }
+
+        #[pallet::weight(<T as pallet::Config>::WeightInfo::set_checkpoint_fee())]
+        #[pallet::call_index(6)]
+        pub fn set_checkpoint_fee(
+            origin: OriginFor<T>,
+            chain_id: ChainId,
+            new_fee: BalanceOf<T>,
+        ) -> DispatchResult {
+            ensure_root(origin)?;
+
+            CheckpointFee::<T>::insert(chain_id, new_fee);
+            Self::deposit_event(Event::CheckpointFeeUpdated { chain_id, new_fee });
+
+            Ok(())
+        }
     }
 
     impl<T: Config> Pallet<T> {
+        pub(crate) fn charge_fee(handler: T::AccountId, chain_id: ChainId) -> DispatchResult {
```

Comment:
when a different token than the native is used to charge a fee for a checkpoint of a chain, does that mean that we expect to pay 0 amount of that token or standard fees plus the checkpoint fee as a tip?
What happens in the case of `submit_checkpoint_with_identity`?

Reply:
We pay fees in the native token only with the current implementation

Reply:
So lets assume that the checkpoint fee is set to 1 NativeToken (NT) and that every tx standard_fee os 0.01 NT
```
total_fees = 0.01 NT + 1 NT = 1.01 NT
```
Is that correct?

Reply:
yes